test_e2e/test_app.py

```diff
@@ -10,7 +10,10 @@
 
 @pytest.fixture(scope="module")
 def driver():
-    with webdriver.Chrome() as driver:
+    opts = webdriver.ChromeOptions()
+    opts.add_argument('--headless')
+    opts.add_argument('--no-sandbox')
+    with webdriver.Chrome("./chromedriver", options=opts) as driver:
```

Comment:
It's nicer if chromedriver is on the PATH and you can keep this as `webdriver.Chrome(options=opts)` rather than requiring chromedriver to be in the project folder.